jenkins-scripts/dsl/_configs_/OSRFWinCompilationAnyGitHub.groovy

```diff
@@ -0,0 +1,24 @@
+package _configs_
+
+import javaposse.jobdsl.dsl.Job
+
+/*
+  -> OSRFWinCompilation
+  -> GenericAnyJob
+
+  Implements:
+    - compiler warning
+*/
+class OSRFWinCompilationAnyGitHub
+{
+  static void create(Job job,
+                     String github_repo,
+                     boolean enable_testing  = true,
+                     ArrayList supported_ros_distros = [])
```

Comment:
Here too, is `supported_ros_distros` needed?

Reply:
passed the parameter to GenericABIAnyGitHub too.

Reply:
I don't think it's used yet, but maybe someday. For now, it's just a pass-through.